app/static/ie-status.json

```diff
@@ -2934,7 +2934,7 @@
     "name": "Number Built-ins (ES6)",
     "category": "JavaScript",
     "summary": "New methods and properties on Number, including isFinite, isInteger, isSafeInteger, isNaN, EPSILON, MIN_SAFE_INTEGER, and MAX_SAFE_INTEGER.",
-    "link": "https://people.mozilla.org/~jorendorff/es6-draft.html",
+    "link": "http://ecma-international.org/ecma-262/6.0/index.html",
```

Comment:
Is there a hash nav we can include to jump to the specific feature (rinse and repeat this for the lines below)?